src/translations/errorpage.translations.js

```diff
@@ -1,10 +1,44 @@
 export const ERROR_PAGE_MESSAGE = {
-  0: {
-    title: 'Щось пішло не так...',
-    toHomepage: 'На головну'
-  },
-  1: {
-    title: 'Something went wrong...',
-    toHomepage: 'Go to homepage'
-  }
+  BUSINESS_TEXT_NOT_FOUND: [
+    { value: 'текст бізнесу не знайдений' },
+    { value: 'business text not found' }
```

Comment:
I think the message has to start with a capital letter

Reply:
fixed